rules/windows/defense_evasion_defender_disabled_via_registry.toml

```diff
@@ -93,10 +93,14 @@ registry where host.os.type == "windows" and event.type in ("creation", "change"
    )
   ) and
 
-  not process.executable :
-           ("?:\\WINDOWS\\system32\\services.exe",
-            "?:\\Windows\\System32\\svchost.exe",
-            "?:\\Program Files (x86)\\Trend Micro\\Security Agent\\NTRmv.exe")
+  not
+    (
+      process.executable : (
+          "?:\\WINDOWS\\system32\\services.exe",
+          "?:\\Windows\\System32\\svchost.exe",
+          "?:\\Program Files (x86)\\Trend Micro\\Security Agent\\NTRmv.exe"
+      ) and user.id : "S-1-5-18"
```

Comment:
I think for this one, it might be better to not exclude SYSTEM, given that this is something adversaries might do after gaining SYSTEM onto a local client. WDYT?

Reply:
This adds the condition that the executables above must run as SYSTEM to be excluded. So, we are restricting a bit more the condition